src/components/CouponInput/CouponInputStyled.js

```diff
@@ -84,6 +85,12 @@ export const InputElementStyled = styled.input.attrs(() => ({
         `}
     `}
 
+  ${isRTL() &&
```

Comment:
Or you could do the same as you did with the paddings & margins and set the `inset-inline-start` property to `37px` and get rid of the `right`s and `left`s in here and in :74